### PR TITLE
Desktop view

### PR DESCRIPTION
**Pull request by mrprotocoll**
Add desktop view
Update README

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -200,25 +207,35 @@ <h5>Skills</h5>
...
+                            <input type="email" required name="@gmail.com" placeholder="Please enter your email"/> 
+                        </div>
+                        <div class="form-input">
+                            <textarea required name="message" placeholder="Write your message here" maxlength="500"></textarea>
+                        </div>
+                        <div class="contact-btn">
+                            <button class="btn">Get It Touch</button>
+                        </div>
+                        
+                    </form>
```
Comment (mckent05) [inline index.html:235]: - Good job on the work done so far, however this section of your portfolio has some minor differences from what is in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your design:
-- Please note the marked section in the image of the Figma design I attached below, I suggest you implement the missing design 🙏 
yhryrt

Figma Design:
rynyth

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @mrprotocoll,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there


## Highlights
- All linter checks are passing ✔️ 
- The mobile and desktop version of your portfolio looks impressive 👌 
- You have used the right workflow for this milestone 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mrprotocoll) [reply index.html:235]: Thank you, dunno how that fell off, I've just made the changes

**Review by jerryowusu — APPROVED** [review]
Hi @mrprotocoll , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve


## Highlights
All linters are passing :heavy_check_mark: 
Used correct GitHub flow :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**